processor/error/package_tests/foo

```diff
@@ -0,0 +1,3 @@
+git mv TestProcessErrorMininmalPayloadException.approved.json TestProcessErrorMinimalPayloadException.approved.json
+git mv TestProcessErrorMininmalPayloadLog.approved.json TestProcessErrorMinimalPayloadLog.approved.json
+git mv TestProcessErrorMininmalService.approved.json TestProcessErrorMinimalService.approved.json
```

Comment:
Was it intended to commit this file?

Reply:
oops, rebasing that out.  Those spelling fixes were tricky.